app/models/participant.rb

```diff
@@ -22,4 +23,21 @@ def currency
     self.paid = false if price_changed?
     write_attribute(:price, nil) if activity.price == self.price
   end
+
+  # Update logs before deleting a Participant to keep what happened
+  def rewrite_logs_before_delete!
```

Comment:
The result from this function is not as expected. The object now only shows the action (eg. "activities" for updating activity and "participants" when adding a participant) and the action is empty except when adding a user.
![Screenshot from 2020-04-16 14-36-03](https://user-images.githubusercontent.com/16525696/79456926-a166c680-7fef-11ea-8b14-da7ddc732c8a.png)